core/templates/dev/head/dashboard/notifications_dashboard.html

```diff
@@ -26,46 +26,59 @@ <h2 class="oppia-page-heading">Notifications</h2>
-              <span ng-if="notification.type === 'exploration_commit'"
-                    class="glyphicon glyphicon-flag"
-                    title="Exploration commit"></span>
+        <md-card ng-repeat="notification in recentNotifications"
+          ng-class="{'oppia-dashboard-row-recent': lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername, 'oppia-dashboard-row': !lastSeenMsec || lastSeenMsec >= notification.last_updated_ms || notification.author_username === currentUsername, '.oppia-exploration-commit-notification': lastSeenMsec && lastSeenMsec < notification.last_updated_ms}" ng-click="navigateToItem(notification.activity_id, notification.type)">
+          <div>
+            <div class="oppia-notification-type">
               <span ng-if="notification.type === 'feedback_thread'"
                     class="glyphicon glyphicon-comment"
                     title="Feedback thread"></span>
-            </td>
-            <td><[notification.activity_title]></td>
-            <td><[getLocaleAbbreviatedDatetimeString(notification.last_updated_ms)]></td>
-            <td><[notification.author_username]></td>
-            <td><[notification.subject]></td>
-          </tr>
-        </table>
+              <span ng-if="notification.type === 'exploration_commit'"
+                    class="glyphicon glyphicon-pencil"
+                    title="Exploration commit"></span>
+            </div>
+            <div ng-if="notification.type === 'exploration_commit'">
+              <div class="oppia-notification-author-username">
+                <strong>
+                  <a ng-click="navigateToProfile($event, notification.author_username)"><[notification.author_username]></a>
+                  <span ng-if="!notification.author_username">
+                    Anonymous
+                  </span>
+                </strong>
+                made a new commit to
+                <span class="oppia-notification-activity-title">
+                  <[notification.activity_title]>
+                </span>
+              </div>
+            </div>
+            <div ng-if="notification.type === 'feedback_thread'">
+              <div class="oppia-notification-author-username">
+                <strong>
+                  <a ng-click="navigateToProfile($event, notification.author_username)"><[notification.author_username]></a>
+                  <span ng-if="!notification.author_username">
+                    Anonymous
+                  </span>
+                </strong>
+                left a comment on
+                <span class="oppia-notification-activity-title">
+                  <[notification.activity_title]>
+                </span>
+              </div>
+            </div>
+            <div class="oppia-notification-last-updated">
+              <[getLocaleAbbreviatedDatetimeString(notification.last_updated_ms)]>
+            </div>
+            <div class="clearfix"></div>
```

Comment:
Still not keen on this. What about creating two divs inside each card, one vertically above the other? In the first div, display everything except the date; in the second, display only the date (and float it right).